pipelines/common/src/test/resources/sql-on-fhir-v2-test-result.json

```diff
@@ -0,0 +1,6951 @@
+[{
```

Comment:
nit: Should we document in this file, on how this is generated? I see that it is documented in the `sql-on-fhir-v2-tests/README.md` file, but felt it would be good to document here as well.

Reply:
Ok, understood.